opts/options.go

```diff
@@ -15,8 +15,9 @@ import (
 const ProtocolDHTOld protocol.ID = "/ipfs/dht"
 
 var (
-	ProtocolDHT      protocol.ID = "/ipfs/kad/1.0.0"
-	DefaultProtocols             = []protocol.ID{ProtocolDHT}
+	ProtocolDHT            protocol.ID = "/ipfs/kad/2.0.0"
+	DefaultProtocols                   = []protocol.ID{ProtocolDHT, "/ipfs/kad/1.0.0"}
+	DefaultClientProtocols             = []protocol.ID{ProtocolDHT}
```

Comment:
@aschmahmann Thanks for the great offline explanation. After you do clean & un-export these, please do add good documentation to them.